book_source/05_developer_guide/03_git/01_Using-Git.Rmd

```diff
@@ -109,79 +111,51 @@ You can setup SSH keys to make it easier to commit cod back to GitHub. This migh
 ### During development:
 
 * commit often; 
-* each commit can address 0 or 1 issue; many commits can reference an issue (see [[Link commits to issue|Using-Git#link-commits-to-issues]])
-* ensure that all tests are passing before anything is pushed into master.
-
-#### Basic Workflow
```

Comment:
This section should not be deleted. It's not the recommended workflow, but it is a useful conceptual stepping stone for those first learning.

Reply:
I just moved it down the page did not delete it. I've put it back though near the top.